File: c/deepa2/surekha1/WindowsApplication1/WindowsApplication1/Form1.vb
Public Class Form1
    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Dim s, r, t, n As Integer
        s = 0
        r = 0
        t = 0
        n = TextBox1.Text
        t = n
        While (n <> 10)
            r = n Mod 10
            s = (s * 10) + r
            n = n \ 10
        End While
        If (s = t) Then
            MessageBox.Show("The Number is Palindrome Number")
        Else
            MessageBox.Show("The Number is not a Palindrome Number")
        End If
    End Sub
End Class

File: c/deepa2/SUREKHA/fileio5/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace fileio5
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            string fname = "g://prashant/test3.txt";
            string textLine = " ";
            if (System.IO.File.Exists(fname) == true) //file check
            {


                System.IO.StreamReader objReader;
                objReader = new System.IO.StreamReader(fname);

                do
                {
                    textLine = textLine + objReader.ReadLine() + "\r\n";
                    //MessageBox.Show("coming up one line:");
                    textBox1.Text = textLine;

                } while (objReader.Peek() != -1); // peek = teal the end of the file 

                textBox1.Text = textLine;
                objReader.Close();
            }
            else
            {

                System.IO.FileAccess objReader1;
                objReader1 = new System.IO.FileAccess();
               MessageBox.Show("file does not exist:");
            }


        }
    }
}
